pkg/cli/adbtunnel.go

```diff
@@ -0,0 +1,231 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"net"
+	"sync/atomic"
+
+	wclient "github.com/google/cloud-android-orchestration/pkg/webrtcclient"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pion/webrtc/v3"
+	"github.com/spf13/cobra"
+)
+
+const hostFlag string = "host"
```

Comment:
nit: could this be moved to cli.go, it's going to be used by `cvd` subcommand as well?

Reply:
done